WordPress/WordPressTest/BlogListViewModelTests.swift

```diff
@@ -70,4 +70,120 @@ final class BlogListViewModelTests: CoreDataTestCase {
         let displayedNames = viewModel.allSites.map(\.title)
         XCTAssertEqual(displayedNames, [".Org", "51 Zone", "A", "C"])
     }
+
+    func test_topSites_includes_current_site() throws {
```

Comment:
(nit) please use camel case. Btw, for new tests (new suites), feel free to use Swift Testing.

Reply:
Updated here 8aeec842f646a7e832e43cda0cbc0e28aba46d41 👍